SDK/Utils/Extensions/StringExtension.swift

```diff
@@ -14,8 +14,10 @@ extension String {
     func webcalURL() -> URL? {
         let urlBegin = self.range(of: "://")!.upperBound
         let urlString = self[urlBegin..<self.endIndex]
-        let webcal = URL(string: "webcal://\(urlString)")
-        return webcal
+        
+        var urlComponents = URLComponents(string: "webcal://\(urlString)")
+        urlComponents?.queryItems?.append(URLQueryItem(name: "u", value: Config.uuid))
```

Comment:
What does this code do if there are no query parameters yet? Does it append anything or does it just "skip" the statement?

Reply:
@dmfs It always adds the u and uuid, but it only runs when we need to format a url to a webcal